tfhe/src/integer/server_key/mod.rs

```diff
@@ -14,6 +14,7 @@ use serde::{Deserialize, Serialize};
 
 /// Error returned when the carry buffer is full.
 pub use crate::shortint::CheckError;
+pub use radix_parallel::{MiniUnsignedInteger, Reciprocable};
```

Comment:
we should we do the same for the other traits introduced recently ?

Reply:
We could